ghost/i18n/locales/it/newsletter.json

```
@@ -1,24 +1,24 @@
 {
-    "By {authors}": "",
-    "Comment": "",
-    "complimentary": "",
-    "Email": "",
-    "free": "",
-    "Keep reading": "",
-    "Less like this": "",
-    "Manage subscription": "",
-    "Member since": "",
-    "More like this": "",
-    "Name": "",
+    "By {authors}": "di",
+    "Comment": "Commenta",
+    "complimentary": "Omaggio",
```

this word will be in the middle of the sentence, so lowercase, please.
```suggestion
    "complimentary": "omaggio",
```